src/main.ts

```diff
@@ -25,6 +25,23 @@ export {
  */
 export { RobotClient as Client } from './robot';
 
+/**
+ * @deprecated Use {@link RobotClient} instead.
```

Comment:
why is this deprecated?

Reply:
I was just following the RobotClient above.